src/applications/disability-benefits/all-claims/utils/form0781.js

```diff
@@ -40,3 +40,10 @@ export function isCompletingForm0781(formData) {
       form0781WorkflowChoices.COMPLETE_ONLINE_FORM
   );
 }
+
+export function isRelatedToMST(formData) {
+  return (
+    isCompletingForm0781(formData) &&
+    formData.mentalHealth.eventTypes.mst === true
```

Comment:
Previously overlooked that optional chaining should be added here to avoid throwing a TypeError, in the event a property is null or undefined.